src/components/editor/EditorAndList.tsx

```diff
@@ -17,22 +18,27 @@ function EditorAndList(props: Props) {
     return (
         <Grid
             container
-            spacing={2}
             sx={{
                 bgcolor: 'background.paper',
-                height: heightVal,
+                height: `${heightVal + DEFAULT_TOOLBAR_HEIGHT}px`,
+                mb: 2,
```

Comment:
Need to make sure the things using editor know about the "toolbar" at the top now. Also adding some margin below to make sure there is a little empty space around the editor. Having editor near the button of the page without extra space just looked odd to me.